py3status/modules/xrandr.py

```diff
@@ -119,6 +118,7 @@ class Py3status:
     """
     """
     # available configuration parameters
+    background_command = None
```

Comment:
(Suggestion) Go with a generic option... Use `command = None` instead to signify that this can be used for other things too, not just a background setter.

Reply:
Agreed. The may be other uses for having a command run after the display configuration changes.